bookie_ml.py

```diff
@@ -0,0 +1,372 @@
+import numpy as np
+
+import nltk
+nltk.download('stopwords')
+from nltk.corpus import stopwords
+stop_words = set(stopwords.words('english'))
+nltk.download('averaged_perceptron_tagger')
+from urllib.request import urlopen
+from bs4 import BeautifulSoup
+nltk.download('punkt')
+nltk.download('wordnet')
+from nltk.corpus import wordnet as wn
+
+def synonymous(word1):
```

Comment:
Since Bookie requires Python 3.6+, try to use type hinting. More information here https://realpython.com/python-type-checking/#annotations